experiments/ClimaEarth/user_io/postprocessing.jl

```diff
@@ -0,0 +1,123 @@
+## helpers for user-specified IO
+include("debug_plots.jl")
+include("diagnostics_plots.jl")
+
+"""
+    postprocess_sim(mode_type::AbstractSlabPlanetModeType, cs, postprocessing_vars)
+
+If they exist, perform conservation checks, for any slabplanet simulation, and then call
+`common_postprocessing` to perform common postprocessing tasks that are common to all simulation types.
+"""
+function postprocess_sim(mode_type::AbstractSlabPlanetModeType, cs, postprocessing_vars)
+    (;
+        dir_paths,
+        plot_diagnostics,
```

Comment:
The `postprocess_sim` methods don't need `plot_diagnostics`, only `common_postprocessing` does. It would be good to not retrieve it in methods it isn't needed, and we should check that all the other variables are needed too